apps.json

```
@@ -494,7 +494,7 @@
     },
     "cryptpad": {
         "category": "office",
-        "level": 8,
+        "level": 2,
```

```suggestion
        "level": 8,
```

cf. last CI check: https://ci-apps.yunohost.org/ci/job/4479